docs/source/usecases/homesale-forecast.rst

```diff
@@ -0,0 +1,135 @@
+To load the home sales data into your database, see the complete `home sale forecasting notebook on Colab <https://colab.research.google.com/github/georgia-tech-db/eva/blob/staging/tutorials/16-homesale-forecasting.ipynb>`_.
+
+Preview the Home Sales Data
+-------------------------------------------
+
+We use the `raw_sales.csv of the House Property Sales Time Series <https://www.kaggle.com/datasets/htagholdings/property-sales?resource=download>`_ in this usecase. The data contains five columns: postcode, price, bedrooms, datesold, and propertytype.
+
+.. code-block:: sql
+
+   SELECT * FROM postgres_data.home_sales LIMIT 3;
+
+This query previews the data in the home_sales table:
+
+.. code-block:: 
+
+    +---------------------+------------------+---------------------+---------------------+-------------------------+
+    | home_sales.postcode | home_sales.price | home_sales.bedrooms | home_sales.datesold | home_sales.propertytype |
+    |---------------------|------------------|---------------------|---------------------|-------------------------|
+    |                2607 |           525000 |                   4 |          2007-02-07 |                   house |
+    |                2906 |           290000 |                   3 |          2007-02-27 |                   house |
+    |                2905 |           328000 |                   3 |          2007-03-07 |                   house |
+    +---------------------+------------------+---------------------+---------------------+-------------------------+
+
+Train a Home Sale Forecasting Model
+-----------------------------------
+
+Let's next train a time-series forecasting model from the home_sales table using EvaDB's ``CREATE FUNCTION`` query.
+Particularly, we are interested in the price of the properties that have three bedrooms and are in the postcode 2607 area.
+
+.. code-block:: sql
+
+  CREATE FUNCTION IF NOT EXISTS HomeSaleForecast FROM
+    (
+      SELECT propertytype, datesold, price
+      FROM postgres_data.home_sales
+      WHERE bedrooms = 3 AND postcode = 2607
+    )
+  TYPE Forecasting
+  PREDICT 'price'
+  TIME 'datesold'
+  ID 'propertytype'
+  FREQUENCY 'W';
+
+While running this query, EvaDB will train different models respectively for different property types (i.e., house, unit). Here is the query's output ``DataFrame``:
+
+.. note::
+
+   Go over :ref:`forecast` page on fine tuning the forecast model.
+
+.. code-block:: 
+
+   +----------------------------------------------+
+   | Function HomeSaleForecast successfully added |
+   +----------------------------------------------+
+
+Predict the Home Price using the Trained Model
+----------------------------------------------
+
+Next we use the trained ``HomeSaleForecast`` to predict the home sale price for next 3 weeks.
+
+.. code-block:: sql
+
+   SELECT HomeSaleForecast(3);
+
+The input of the trained model is the horizon, the steps we want to forecast in the future. Here is the query's output ``DataFrame``:
```

Comment:
Can you connect it back to the parameters used in the training? Like because we specify frequency as`W, so the provided horizon represents weeks (correct me if I'm wrong).

Reply:
This is one issue I summarized for the current forecasting. The frequency does not affect the horizon. @americast Is this normal for statsforecasting? For now, I update the docs to make it clear it is 3 week.

Reply:
Yes, frequency has nothing to do with horizon, for both the libraries.

Reply:
@americast How is the horizon decided? Say I want to forecast the price in months instead of weeks. Is it possible?

Reply:
Will discuss with Sayan offline as a limitation of the current Forecasting model, and how to improve. Close this thread now.